checkov/terraform/graph_builder/utils.py

```diff
@@ -26,9 +26,9 @@
 def is_local_path(root_dir: str, source: str) -> bool:
     # https://www.terraform.io/docs/modules/sources.html#local-paths
     return (
-        source.startswith("./")
-        or source.startswith("/./")
-        or source.startswith("../")
+        source.startswith(r"./")
```

Comment:
this change is to avoid spam of debug warning about using `/` in string being deprecated

Reply:
nice, I was always curious, where those warnings came from 😄